src/styles/contact-us.scss

```diff
@@ -0,0 +1,115 @@
+.contact-us {
+  font-family: 'Space Grotesk',sans-serif;
+  color: $main-color;
+
+  @include fadeIn($appear-duration, $delay-contact-us);
+
+  &__title {
+    font-size: 32px;
+    line-height: 32px;
+  }
+
+  &__form {
+    margin-top: 48px;
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+  }
+
+  &__input {
+    height: 48px;
+    padding-inline: 16px;
+    color: #7E7E83;
+
+    font-family: 'Space Mono',sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+
+    border-radius: 8px;
+    background-color: #F8F8FA;
+    border:none;
+
+    &::placeholder {
+      left: 16px;
+      color: $input-font-default-color;
+
+      font-family: 'Space Mono', sans-serif;
+      font-size: 14px;
+    }
+
+    &:hover {
+      &::placeholder {
+        color: $main-color;
+      }
+    }
+
+    &:focus {
+      border: 1px solid $main-color;
+      color: $main-color;
+
+      &::placeholder {
+        color: $main-color;
+        border-color: $main-color;
+        font-size: 14px;
+      }
+    }
+  }
+
+  &__text-area {
+    padding: 14px 16px;
+
+    color: $input-font-default-color;
+    font-family: 'Space Mono', sans-serif;
```

Comment:
use variable 
You can write this font for the parent element, and then - inherit it for child elements.